Code/Engine/Audio/AudioMetadataUtils.hpp

```diff
@@ -10,4 +11,5 @@
 class Texture;
 
 Texture* GetImageFromFileMetadata(const std::wstring& fileName);
-bool IncrementPlaycount(const std::wstring& fileName);
\ No newline at end of file
+bool IncrementPlaycount(const std::wstring& fileName);
+std::vector<std::wstring> GetSupportedFiles(const std::wstring& folder);
```

Comment:
I'd change the name to GetSupportedAudioFiles, since that makes it more apparent that we're dealing with audio here.